### PR TITLE
Additional Info Tabs Added

### PR DESCRIPTION
**Pull request by FeyFaelis**
I've added a good amount of them now. Smile.

### CODE REVIEW

File: Marketplace.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The Market Place is where you'll go in order to sell and buy other items from players. You can get here by clicking Marketplace on the side or clicking on your gold bag above your stockpile. You can see all the various items within the game here that other players are selling. When you click on any item, it'll open a different window that will show the different listings by other players. By default it is listed by price, lowest to highiest. To return, you can click the "Back" button in the top left.
```
Comment (IceFreez3r) [inline Marketplace.md]: > By default it is *~~listed~~ordered* by price, lowest to *high~~i~~est*.

File: PlatShop.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Platinum (Plat) can be purchased with real life money in order to obtain global player buffs, quality of life tokens, and cosmentic items.
```
Comment (IceFreez3r) [inline PlatShop.md]: cosmentic

File: PlatShop.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Platinum (Plat) can be purchased with real life money in order to obtain global player buffs, quality of life tokens, and cosmentic items. 
+<br />
+Global player buffs will apply to the entire server for others to enjoy, as well as a special thank you in general chat when purchased. You can opt out of displaying your name by clicking "Anonymous Purchase". 20 Plat is equal to one minute of buff time. Buffs start out at level 2, but can upgrade to level 3 and 4 once certain time threshholds are met.
```
Comment (IceFreez3r) [inline PlatShop.md]: Level 3 is 1 day. Level 4 is 1 week. Once the threshold is reached, the buff stays at that level including later extensions by different players or yourself

File: Settings.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Equipped Comparison: Weither to show your currently equipped items when you hover over an item that goes into that equip slot.
+<b> Privacy </b>
+Anonymous: If you want your name to be displayed or not in Community Data Releases.
+<b> Misc </b>
+Auto Vault Like Items: If you want items you obtained to automatically be put in the vault if they are already in your vault.
+Auto Tacklebox Like Items: If you want bait to automatically be put in your tacklebox if they are already in there.
+Use Legacy Sorting: If you want to sort items the way they used to be sorted. (This will ignore other settings)
+Sort by Value before ID: If you want to sort items by value instead of their item ID's
+High Performance Mode: Will disable some animations and graphics to prioritize Performance.
+Enable Pinned Tabs: If you want to pin tabs for quicker access.
```
Comment (IceFreez3r) [inline Settings.md]: Would prefer if we have a more general description here. Otherwise it will be outdated in a week :D

File: Shrine.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+The Shrine is an optional activity that all players can access and benefit from weekly. Every week, the shrine will select random buffs that players can offer gold towards to obtain. There can be up to three different buffs that can be selected at random. Each tier of the shrine has an offering goal, that once reached, will be applied at the end of the week. The duration milestones will increase how long the buffs are active, while the buff milestones will give one level of that buff for the duration. This means that if there is five of a single buff within the shrine, you will get five levels of that buff for the duration. (Like Haste 5 for example.) 
+<br />
+In order to make an offering, input the amount of gold you wish to offer in the text box below, then click "OFFER" to confirm. 
+<br />
+<b> Buffs that can appear </b>
+(List of buffs that can appear within the shrine so players can stop asking.)
```
Comment (IceFreez3r) [inline Shrine.md]: Likely to be forgotten aswell, but at least the buffs here don't change a lot

File: Marketplace.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+The Market Place is where you'll go in order to sell and buy other items from players. You can get here by clicking Marketplace on the side or clicking on your gold bag above your stockpile. You can see all the various items within the game here that other players are selling. When you click on any item, it'll open a different window that will show the different listings by other players. By default it is listed by price, lowest to highiest. To return, you can click the "Back" button in the top left. 
+<br />
+<b> Selling </b>
+In order to sell items on the market, you can click the "Sell" button also in the top left on the main screen. Here you will see all the items you can sell, as well as items you already have listed on the market if any. When you click on an item, it will bring up a screen asking you to select the amount you wish to sell of that item, and for how much. Once you are satisfied with your listing, click "Create Listing" and your item will now be on the market for other players to purchase with gold should they choose to do so.
```
Comment (IceFreez3r) [inline Marketplace.md:4]: Somethink similar to this
Tip: You can use K,M,B in the price input. E.g. 2.5M for 2,500,000
would be nice.

File: Settings.md (hunk 4 above)
Comment (FeyFaelis) [reply Settings.md]: Fair

Comment (FeyFaelis) [reply Settings.md]: I'll move the setting info to a separate note file just in case.

File: Shrine.md (hunk 5 above)
Comment (FeyFaelis) [reply Shrine.md]: I'll remove it then, just some food for thought.

**Comment by FeyFaelis** [comment]
Fixed and added based upon comments made.

**Comment by FeyFaelis** [comment]
Updated guides based on community feedback

**Pull request merged**